frontstage_api/resources/surveys/upload_collection_instrument.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from flask import request, Response
+from flask_restplus import Resource
+from structlog import wrap_logger
+
+from frontstage_api import api, app, auth
+from frontstage_api.controllers import case_controller, collection_instrument_controller
+from frontstage_api.exceptions.exceptions import FileTooLarge
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+@api.route('/upload-ci')
+class UploadCollectionInstrument(Resource):
+
+    @staticmethod
+    @auth.login_required
+    def post():
+        case_id = request.args['case_id']
+        party_id = request.args['party_id']
+        logger.info('Attempting to upload collection instrument', case_id=case_id, party_id=party_id)
+
+        if request.content_length > app.config['MAX_UPLOAD_LENGTH']:
```

Comment:
Might be better to have this check done in the front end?

Reply:
Yeah good point I'll add that, should we have it in both maybe?

Reply:
Don't think there's a need to have them in both